README.md

```diff
@@ -14,8 +14,35 @@ This is a web interface for [OwnTracks](https://github.com/owntracks/recorder),
 
 ## Installation
 
+### Manual install
+
 Clone the repository and copy `index.html` and the `static/` directory to your server's webroot. The API is expected to be reachable under the same domain as the web interface.
 
+### Docker
+
+You can launch directly via docker run like this :
+```bash
+docker run -d -p 80:80 -e SERVER_URL=otrecorder-url -e SERVER_PORT=otrecorder-port cosme/owntracks-ui
+```
```

Comment:
Let's use `console` (see https://stackoverflow.com/a/49004070) here, and add a prompt as well as changing `SERVER_URL` to `SERVER_HOST` - like this:

````
```console
$ docker run -d -p 80:80 -e SERVER_HOST=otrecorder-url -e SERVER_PORT=otrecorder-port cosme/owntracks-ui
```
````

Reply:
Done :)